yaksh/views.py

```diff
@@ -794,7 +794,8 @@ def courses(request):
         raise Http404('You are not allowed to view this page')
     courses = Course.objects.filter(creator=user, is_trial=False)
     allotted_courses = Course.objects.filter(teachers=user, is_trial=False)
-    context = {'courses': courses, "allotted_courses": allotted_courses}
+    context = {'courses': courses.order_by('-id'), 
```

Comment:
<code>order_by</code> should not be done here in a dictionary. Do it in the line above with course filter statement.